docs/reference/tab-widgets/troubleshooting/data/increase-node-capacity.asciidoc

```diff
@@ -0,0 +1,299 @@
+    }
+  }
+}
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
+
+<1> Represents a comma separated list of data tier node roles this index is allowed
+to be allocated on, the first one in the list being the one with the higher priority
+i.e. the tier the index is targeting.
+e.g. in this example the tier preference is `data_warm,data_hot` so the index is
+targeting the `warm` tier and more nodes with the `data_warm` role are needed in
+the {es} cluster.
+
+//end::kibana-api-ex-2[]
+
+//tag::increase-azs[]
+[increase-azs]
+Now that you know the tier, increase the number of availability zones.
+
+. Go back to the deployment's landing page by clicking on the three horizontal bars on the top left of the screen
+and choosing **Manage this deployment**. On that page click the **Manage** button, and choose **Edit deployment**.
+
+. In the {es} section, find the tier where the replica shards could not be assigned. Select a higher number
+of availability zones for that tier.
++
+
+[role="screenshot"]
+image::images/data-tiers/ess-advanced-config-data-tiers.png[{kib} Console,align="center"]
+//end::increase-azs[]
+
+If it is not possible to increase the number of availability zones, you can reduce the number of replicas of your index
+data. We'll achieve this by inspecting the <<dynamic-index-number-of-replicas,`index.number_of_replicas`>> index setting
+index setting and decreasing the configured value.
+
+//tag::kibana-api-ex-2[]
+. Access {kib} as described above.
+
+. Inspect the <<dynamic-index-number-of-replicas,`index.number_of_replicas`>> index setting.
++
+[source,console]
+----
+GET /my-index-000001/_settings/index.number_of_replicas
+----
++
+The response will look like this:
++
+[source,console-result]
+----
+{
+  "my-index-000001" : {
+    "settings" : {
+      "index" : {
+        "number_of_replicas" : "2" <1>
+      }
+    }
+  }
+}
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
+
++
+<1> Represents the currently configured value for the number of replica shards
+required for the index
+
+. Use the `_cat/nodes` API to find the number of nodes in the target tier:
++
+[source,console]
+----
+GET /_cat/nodes?h=node.role
+----
+// TEST[continued]
++
+The response will look like this, containing one row per node:
++
+[source,console-result]
+----
+himrst
+mv
+himrst
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
++
+You can count the rows containing the letter representing the target tier to know how many nodes you have. See
+<<cat-nodes-api-query-params>> for details. The example above has two rows containing `h`, so there are two
+nodes in the hot tier.
++
+. <<indices-update-settings, Decrease>> the
+value for the total number of replica shards required for this index. The new value needs to be less
+than the number of nodes found above minus one. Since the example above found 2 nodes in the hot tier,
```

Comment:
```suggestion
. <<indices-update-settings, Decrease>> the value for the total number of 
replica shards required for this index. As replica shards cannot reside on the
same node as primary shards for <<high-availability-cluster-design,high
availability>>, the new value needs to be less than the number of nodes found
above minus one. Since the example above found 2 nodes in the hot tier,
```

I meant to link the high availability text to the following link, is that correct? 
https://www.elastic.co/guide/en/elasticsearch/reference/current/high-availability-cluster-small-clusters.html